tests/fixtures/smart/P110M(AU)_1.0_1.2.3.json

```diff
@@ -124,19 +127,6 @@
     "get_connect_cloud_state": {
         "status": 1
     },
-    "get_energy_usage": {
```

Comment:
Hmm, any ideas why this was moved and reordered? If the ordering is stable, should we enforce the order to help minimizing changes on fixture updates?

Reply:
I think at one stage I added this manually to the fixture but I can't recall why. The `update_fixtures` sorts by keys (same as dump_devinfo) hence why it's now moved.